src/esthri/src/http_server.rs

```diff
@@ -48,7 +48,7 @@ use async_stream::stream;
 
 use futures::stream::StreamExt;
 
-use eyre::{eyre, Report};
+use anyhow::anyhow;
```

Comment:
Switch back to anyhow, it supports backtraces in stable now too and eyre didn't get us the backtraces that we hoped it would (maybe user error?)

Reply:
What about going to thiserror/a hand rolled error enum - anyhow is more for applications. As a consumer of esthri as a library it would be nice to have a fixed set of errors to work with

Reply:
Probably a decent amount of work though

Reply:
I think it's worth doing

Reply:
Done

Reply:
the optional `http_server` module still uses anyhow, but I removed `anyhow` from the main lib module